runtime/doc/options.txt

```diff
@@ -6280,6 +6280,29 @@ A jump table for the options with a short description can be found at |Q_op|.
 	attributes instead of "cterm" attributes. |highlight-guifg|
 	Requires an ISO-8613-3 compatible terminal.
 
+						*'termpastefilter'* *'tpf'*
+'termpastefilter' 'tpf'	string	(default: "")
```

Comment:
Why do we need this option?

Reply:
This is to prevent problems such as `\x1b[201~` being included in the text to be pasted, which would cause the parenthesized paste mode to complete. As far as I know, `xterm` and `mintty` have such an option. Also, although I couldn't find the option in the manual, `tmux` also converts `\x1b` to `^[` when pasting.  

#### man xterm

```
disallowedPasteControls (class DisallowedPasteControls)
               The allowPasteControls resource is normally used to prevent pasting C1 controls, as well as non-formatting C0 controls such as the ASCII
               escape character.  Those characters are simply ignored.  This resource further extends the set of control characters which cannot be pasted,
               converting each into a space.
```

#### man mintty

```
Filtering pasted text (FilterPasteControls=)
         With this setting, pasted text can be filtered for selected control characters which are then replaced by space. Filtering is not applied when
         Control is held while pasting. The option is a comma-separated list of character tags:
```